openstudiocore/resources/model/OpenStudio.idd

```diff
@@ -14527,6 +14544,18 @@ OS:Coil:Heating:Gas,
        \type object-list
        \required-field
        \object-list ScheduleNames
+  A4 , \field Fuel Type
+       \type choice
+       \key Gas
+       \key NaturalGas
+       \key PropaneGas
+       \key Diesel
+       \key Gasoline
+       \key FuelOil#1
+       \key FuelOil#2
+       \key OtherFuel1
+       \key OtherFuel2
+       \default Gas
```

Comment:
The old default in the forward translator was `NaturalGas`, so we might want to make it that here for consistency.